### PR TITLE
Log ECALLs

### PR DESCRIPTION
**Pull request by letmaik**
Same as #1816 but for ECALLs.

### CODE REVIEW

File: host/sgx/calls.c [hunk 1]
```diff
@@ -742,6 +763,13 @@ oe_result_t oe_ecall(
     if (!(tcs = _assign_tcs(enclave)))
         OE_RAISE(OE_OUT_OF_THREADS);
 
+    oe_log(
+        OE_LOG_LEVEL_VERBOSE,
+        "%s 0x%x OE_ECALL: %s\n",
```
Comment (johnkord) [inline host/sgx/calls.c]: We probably want to be consistent with the other PR which changes this line for OCALLs

Comment (letmaik) [reply host/sgx/calls.c]: Done.

**Comment by letmaik** [comment]
@johnkord I aligned it with the other PR.

**Comment by johnkord** [comment]
Thanks Maik!

bors r+

**Pull request merged**